### DISCUSSION TITLE
Avoiding Android app total reupload?

### DISCUSSION THREAD

**Discussion by matrixes** [immich-app/immich#12083] [Q&A]
Is there any way that I can sanely avoid having to reupload everything from scratch when clearing app data and logging back in?

I have previously uploaded my Camera "album" to Immich without any issues. The server has caught every photo (~1000) that the Android app sent.

I had to do a bit of maintenance on my phone and in that I had to uninstall and reinstall Immich (no changes to the server, though).

When I logged into Immich after the reinstall, I selected the Camera album again, and started syncing. It seems like Immich uploads _every_ single photo again which it already has uploaded earlier. I let it finish, but wanted to try and verify what I saw, so I logged out of the app, cleaned the app data and logged in. It did the same thing as before; uploads every photo to the server even if the photo already exists on the server.

I (probably wrongly) assumed that Immich would do any kind of comparison, without having to upload the file to the server first, between the app and the server and if the comparison matches skip the image and move on to the next. 

There's one bit of information that I hope may assist in troubleshooting. Looking at the Backup view in the app, it says in the "Backup" section: 166 assets. When the app backs up photos, this number does not change. However, "Total" and "Remainder" decreases simultaneously.

**Comment by mmomjian** [comment]
This is not currently possible, but it is tracked as an issue in #1553
